Misc/stable_abi.toml

```diff
@@ -2460,3 +2460,7 @@
     added = '3.13'
 [function.PyMapping_HasKeyStringWithError]
     added = '3.13'
+[function.PyUnstable_PerfTrampoline_CompileCode]
+    added = '3.13'
+[function.PyUnstable_PerfTrampoline_SetPersistAfterFork]
+    added = '3.13'
```

Comment:
Just a drive-by comment... I'm not sure why we would want to include unstable functions like these in the stable ABI. Is there a particular reason why they're being added?

Reply:
Sorry for the confusion. I accidentally included these functions in the stable ABI while trying to fix a failing test. I have since removed them.